src/ftva_etl/metadata/utils.py

```diff
@@ -45,3 +45,17 @@ def strip_whitespace_and_punctuation(items: list[str]) -> list[str]:
     # Add space to right-strip of punctuation to handle spaces after punctuation,
     # then explicitly strip square brackets and spaces from resulting string.
     return [item.rstrip(string.punctuation + " ").strip("[] ") for item in items]
+
+
+def cleanup_production_type(production_type: str) -> list[str]:
+    """Cleanup the production_type string to allow series identification.
+
+    In the Filemaker PD, the production_type field is a list of strings,
+    often delimited by carriage returns (\r).
+    This function cleans up the string to allow series identification.
+
+    :param production_type: A string containing the production types.
+    :return: A list representation of the production types.
+    """
+    # Normalize to lowercase and split by carriage returns
+    return production_type.lower().split("\r")
```

Comment:
The individual terms parsed out of `production_type` should be trimmed after splitting.  I found a few dozen records in the FM data dump which have `"NEWS "` with a trailing space.  Didn't see any leading spaces; didn't check the longer terms, but best to trim everything here.  Examples:
```
 ('M158234', 'NEWS \rSHORT'),
 ('M158235', 'NEWS \rNEWSREELS'),
 ('M158267', 'NEWS \rSHORT\rDOCUMENTARY'),
 ('M160877', 'NEWS '),
```